Pinta.Core/Managers/WorkspaceManager.cs

```diff
@@ -276,6 +262,9 @@ public void CloseDocument (
 		Document document)
 	{
 		int index = open_documents.IndexOf (document);
+
+		if (index == -1) return; // TODO: Maybe throw an exception?
```

Comment:
Yes, throwing an exception would be correct here since the document is expected to be in the workspace. I think the only caller is doing `workspace.CloseDocument(workspace.ActiveDocument)` so nothing should rely on the existing behaviour

Reply:
It's now throwing an `ArgumentException`